src/main/common/router/routablePath.ts

```diff
@@ -41,6 +47,10 @@ export class RoutablePath {
     const path = Object.entries(substitutions).reduce((uri: string, substitution: [string, string]) => {
       const [parameterName, parameterValue] = substitution
 
+      if (!isValidParameterValue(parameterValue)) {
+        throw new Error(`Path parameter :${parameterName} is invalid`)
```

Comment:
Maybe would be good to place `'` quotes around `${parameterName}` so that it's easier to see it is blank?

Reply:
Param name should never be blank, we getting param names from names of the object literal properties.

Reply:
Why are you calling if `StringUtils.isBlank` on it if it's never blank?